core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/EpochManager.scala

```diff
@@ -1,54 +1,46 @@
 package edu.uci.ics.amber.engine.architecture.worker
 
-import edu.uci.ics.amber.engine.common.ambermessage.EpochMarker
-import edu.uci.ics.amber.engine.common.rpc.AsyncRPCClient.ControlInvocation
-import edu.uci.ics.amber.engine.common.rpc.AsyncRPCClient
+import edu.uci.ics.amber.engine.common.AmberLogging
+import edu.uci.ics.amber.engine.common.ambermessage.{
+  ChannelID,
+  ChannelMarkerPayload,
+  NoAlignment,
+  RequireAlignment
+}
 import edu.uci.ics.amber.engine.common.virtualidentity.ActorVirtualIdentity
 
 import scala.collection.mutable
 
-class EpochManager {
+class EpochManager(val actorId: ActorVirtualIdentity) extends AmberLogging {
 
   private val epochMarkerReceived =
-    new mutable.HashMap[String, Set[ActorVirtualIdentity]]().withDefaultValue(Set())
+    new mutable.HashMap[String, Set[ChannelID]]().withDefaultValue(Set())
 
   // markers the arrival of an epoch marker,
```

Comment:
do you mean `marks`? also please use java doc instead of line comments.

Reply:
revised